tests/chainer_tests/test_function_node.py

```diff
@@ -869,6 +869,29 @@ def test_length_check(self):
             chainer.grad([y], [x], [None], [None, None])
 
 
+class TestUnchainSplit(unittest.TestCase):
+
+    def test_unchain_split(self):
+        x = chainer.Variable(numpy.arange(4).astype('f').reshape(2, 2))
+        h0, h1 = chainer.functions.split_axis(x, [1], axis=0)
+        y = chainer.functions.sum(h0)
+        z = chainer.functions.sum(h1)
+        w = y + z
+        h0.unchain()
+
+        dy_dh0 = numpy.array([[1., 1.]])
+        dz_dh1 = numpy.array([[1., 1.]])
+        dy_dx = None
+        dz_dx = numpy.array([[0., 0.], [1., 1.]])
+        dw_dx = numpy.array([[0., 0.], [1., 1.]])
+
+        assert numpy.all(chainer.grad([y], [h0])[0].array == dy_dh0)
```

Comment:
Could you use `testing.assert_allclose`, to be ready for adding GPU tests in future?

Reply:
Ok.